docusaurus/docs-internals/brizy-editor/introduction.md

```diff
@@ -3,3 +3,35 @@ sidebar_position: 1
 ---
 
 # Introduction
+
+The environment where users design pages and oversee site settings is referred to as the `editor`. This environment comprises two key segments: the `preview` and the `editor`. Within each segment, there are additional areas that third-party developers can expand.
+
+---
+
+### Using Controls 
+
+Brizy controls are UI components designed to help build toolbars and gather user data. They come in various types to accommodate different needs:
```

Comment:
this text is copied

there is no need to copy any text, just write a short description what each module contains